client/src/locales/en.json

```diff
@@ -67,45 +66,57 @@
-  "authorization.reset-password.heading": "Reset your password",
-  "authorization.reset-password.email-label": "Enter email",
-  "authorization.reset-password.button-content": "Reset password",
-  "authorization.actions.reset": "An email with instructions to reset password has been sent to {data} if it is associated with an EOC account. Your existing password has not been changed.",
-  "authorization.password-recovery-success": "Password successfully updated. Please go to the {link} and sign in to app.",
-  "authorization.pass-recovery-form.heading": "Set new password",
-  "authorization.actions.recovery-password-default-error": "Account does not exist or is not activated.",
+  "user.name": "Name",
+  "user.email": "Email",
+  "user.password": "Password",
+  "user.photo": "Photo",
+  "user.profile": "Profile",
+  "user.profile.account": "Account info",
+  "user.profile.personal-info": "Personal info",
+  "user.profile.contact-info": "Contact info",
+  "user.profile.change-password": "Change password",
+  "user.profile.account-activation": "Account activated on",
+  "user.profile.account-activation-date": "{date} at {time}",
+  "user.auth-box.description": "The best app to track your office inventory.",
+  "user.auth-box.sign-in": "Sign in to app:",
+  "user.auth-box.sign-in-google": "Sign in with Google",
+  "user.auth-box.cta": "Try it out:",
+  "user.auth-box.demo-button": "Live demo",
+  "user.auth-box.sign-up": "Sign up to app:",
+  "user.auth-box.create-account": "Create account",
+  "user.auth.sign-in": "Sign in",
+  "user.auth.sign-up": "Sign up",
+  "user.auth.change": "Change",
+  "user.auth.sign-up.button.resend-link": "Resend link",
+  "user.auth.forgot-password.question": "If you forgot the password please click {link} to reset it.",
+  "user.auth.forgot-password": "Forgot password",
+  "user.auth.input.current-password": "Current password",
+  "user.auth.input.username.invalid": "Please use min 1 and max 32 chars.",
+  "user.auth.input.username.empty": "Please enter your name.",
+  "user.auth.input.email.invalid": "Please enter a valid email address.",
+  "user.auth.input.password.confirm": "Confirm password",
+  "user.auth.input.new-password.confirm": "Confirm new password",
+  "user.auth.input.password.invalid": "Please use min 4 and max 32 characters without spaces.",
+  "user.auth.input.password.not-match": "Passwords do not match.",
+  "user.auth.sign-up.result-link": "sign in/sign up page",
+  "user.auth.reset-password.heading": "Reset your password",
+  "user.auth.reset-password.email-label": "Enter email",
+  "user.auth.reset-password.button-content": "Reset password",
+  "user.auth.pass-recovery-form.heading": "Set new password",
+  "user.actions.fetch-user-details-error": "Failed to fetch your profile data.",
+  "user.actions.logout-fail": "Logout failed.",
+  "user.actions.login": "Login success.",
+  "user.actions.link-expired.recovery-expired": "Password recovery link has expired.",
```

Comment:
I would avoid dashes for translation if you even want to use javascript APIs to extract and use them as `user.actions.link-expired.recovery-expired` is not a valid property chain but `user.actions.link_expired.recovery_expired` is. However, you can still use the bracket notation `user.actions['link-expired']['recovery-expired']`.